uploader.json

```diff
@@ -743,11 +800,12 @@
                 "is beyond advertised capabilities",
                 "kernel: CIFS VFS: Unexpected SMB signature",
                 "kernel: bnx2fc: byte_count",
-                "kernel: megasas: Found FW in FAULT state, will reset adapter.",
+                "kernel: lockd: Unknown symbol register_inet6addr_notifier",
+                "kernel: lockd: Unknown symbol unregister_inet6addr_notifier",
+                "kernel: megasas: Found FW in FAULT state,will reset adapter.",
                 "khash_super_prune_nolock",
                 "megaraid_sas: FW detected to be in faultstate, restarting it",
                 "mode:0x20",
-                "modprobe: FATAL: Error inserting nfsd",
```

Comment:
Was this dropped on purpose?

Reply:
It's updated in https://github.com/RedHatInsights/insights-plugins/pull/271 by Yi Zhang

Reply:
Sounds good, thanks!